windows-containers/productization.md

```diff
@@ -0,0 +1,187 @@
+---
+title: windows-containers-productization
+authors:
+  - "@ravisantoshgudimetla"
+  - "@aravindhp"
+reviewers:
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@crawford"
+  - "@sdodson"
+creation-date: 2019-08-30
+last-updated: 2019-09-03
+status: implementable
+---
+
+# Windows Containers Productization
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The intent of this enhancement is to allow a cluster administrator to add a
+Windows worker node with a prescribed configuration to an OpenShift cluster as a
```

Comment:
nit: s/worker/compute/, e.g. see the old openshift/installer#1330.  Overhauling existing wording it hard, but no reason new stuff can't use the new wording :).

Reply:
I will make that change in all places. So is it `compute Ignition` or `worker Ignition`?

Reply:
Unless the machineconfigpool is changing to compute I think we should leave this as is.

Reply:
@sdodson is your comment against `s/worker/compute/` or just the Ignition case?

Reply:
just in relation to Ignition